app/models/enumeration/AddressJourneyType.scala

```diff
@@ -73,13 +73,15 @@ object AddressJourneyType extends Enumerable.Implicits {
         case EnterPostcodePage(_, _) => Message("residentialAddress.enterPostcode.h1")
         case ChooseAddressPage(_, _) => Message("residentialAddress.chooseAddress.h1")
         case ManualAddressPage(_, _) => Message("residentialAddress.address.h1")
+        case _ => throw new RuntimeException(s"Unknown page for address journey: $whichPage")
```

Comment:
Do we want RuntimeExceptions here or shall we use Options as previously discussed on another ticket / a custom exception?

Reply:
This error will never happen in reality and if we change to option then it will entail quite a lot of other changes in the code, which won't necessarily be for the better. I added this exception just to resolve compile error introduced by latest scala. One way we could get rid of exception would be to have a new abstract Page parent purely for address journeys. Perhaps worth raising a spike to look into this.